README.md

```diff
@@ -468,22 +468,22 @@ Common flow is the same as you would use usual classes:
 2. Add rules to it
 
 ```php
-use Yiisoft\Validator\Attribute\Embedded;
+use Yiisoft\Validator\Rule\Embedded;
 use Yiisoft\Validator\Rule\Count;
 use Yiisoft\Validator\Rule\Each;
 use Yiisoft\Validator\Rule\Number;
 
 final class Chart
 {
     #[Each([
-        new Embedded(Point::class),
+        new Embedded(),
     ])]
     private array $points;
 }
 
 final class Point
 {
-    #[Embedded(Coordinates::class)]
+    #[Embedded]
```

Comment:
It should also work with specifying a class because the value may be a scalar value.

Reply:
`Embedded` rule for scalar value? Why? 
Can you describe example more detailed?